server/intelligence-service/app/main.py

```diff
@@ -30,23 +26,11 @@ class ChatResponse(BaseModel):
     summary="Start and continue a chat session with an LLM.",
 )
 async def chat(request: ChatRequest):
-    if request.thread_id is None:
-        # Start a new chat session
-        result = start_chat_function(request.message)
-        thread_id = result["thread_id"]
-        state = result["state"]
-        response_message = result["response"]["messages"][-1].content
-        conversations[thread_id] = state
-        return ChatResponse(thread_id=thread_id, response=response_message)
-    else:
-        thread_id = request.thread_id
-        # Check if the thread_id exists
-        if thread_id not in conversations:
-            raise HTTPException(status_code=404, detail="Thread ID not found")
-        state = conversations[thread_id]
-        user_input = request.message
-        result = chat_function(thread_id, user_input, state)
-        state = result["state"]
-        response_message = result["response"]["messages"][-1].content
-        conversations[thread_id] = state
-        return ChatResponse(response=response_message)
+    state = {"messages": []}
+
+    result = send_message(
+        thread_id=request.session_id, input_message=request.message_content, state=state
```

Comment:
`thread_id` is now `session_id` am I right?

Reply:
Yes, I would change the parameter name in the model for the consistency as well